src/include/libutil.h

```diff
@@ -257,6 +257,16 @@ char **break_comma_list(char *list);
  */
 char **break_delimited_str(char *list, char delim);
 
+/*
+ * find index of str in strarr
+ */
+int find_string_ind(char **strarr, char *str);
+
+/*
+ *	find_string - find a string in a NULL terminated string array
+ */
+int find_string(char **strarr, char *str);
```

Comment:
The name find_string() sounds quite misleading (especially if it has to become a generically useful function to the rest of PBS). It sounds like it is a replacement for strstr(). However, it finds a string in an array of strings. Could we rename it to something better?